lib/contentful_rails/engine.rb

```diff
@@ -50,6 +50,15 @@ class Engine < ::Rails::Engine
       end
     end
 
+    initializer "add_contentful_mime_type" do
+      content_type = "application/vnd.contentful.management.v1+json"
+      Mime::Type.register "application/json", :contentful_json, [content_type]
```

Comment:
yep!  the symbol that's passed in here is used to create the mime_type constant.   so their current implementation overrides the existing `JSON` constant set by rails.   this way, it's not overriding the `:json` key, we're setting a new constant.  So no need to unregister